lib/routes/qbitai/category.ts

```diff
@@ -36,19 +38,54 @@ async function handler(ctx) {
 
     const items = feed.items.map((item) => ({
         title: item.title,
-        pubDate: parseDate(item.pubDate),
+        pubDate: parseDate(item.pubDate ?? ''),
         link: item.link,
         author: '量子位',
         category: item.categories,
-        description: item['content:encoded'],
+        // 初始化 description 字段
+        description: '',
     }));
 
+    // 使用 Promise.all 并发处理所有项目
+    await Promise.all(
+        items.map(async (item) => {
+            try {
+                // 尝试从缓存中获取 description
+                const cachedDescription = await cache.tryGet(`description:${item.link}`, async () => {
+                    const response = await ofetch(item.link);
+                    const $ = load(response);
+                    // 获取文章内容，这里假设文章内容在 .article 类中
+                    const articleHtml = $('.article').html();
+                    return articleHtml || '';
+                });
+
+                // 设置 description 字段
+                item.description = cachedDescription;
+            } catch {
+                item.description = '内容获取失败';
+            }
+        })
+    );
```

Comment:
The resolved promises must be passed to `Route['handler']['item']`; otherwise, RSSHub will not cache the response properly.